root/static/partials/c_instantanswer.html

```diff
@@ -0,0 +1,24 @@
+<!-- Outside is wrapped within <div class="ia..."> -->
+
+<div class="grid-6">
+  <strong><a href="https://duck.co/ia/view/{{ia.id}}" ng-click="addImg('iapage', ia.id)">{{ia.name}}</a></strong>
```

Comment:
@jagtalon I just tried to click through to Ben's Markdown Reference IA and it redirected me to the IA index. Maybe the page doesn't exist? maybe the url is being built incorrectly?
<img width="1243" alt="screen shot 2016-04-12 at 3 53 31 pm" src="https://cloud.githubusercontent.com/assets/4480279/14473580/d3f3ae9e-00c6-11e6-8e13-21e5aacb1f05.png">

http://jag.duckduckgo.com:5002/u/guiltydolphin ( Markdownreference is the last IA in his maintaining list)